### PR TITLE
Synchronise mxnet-operator and tf-operator

### PR DESCRIPTION
**Pull request by KingOnTheStar**
Migrate new commits from tf-operator to mxnet-operator (from Sep 3 to Oct 28, 2018)

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/mxnet-operator/13)

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (gaocegege) [inline .idea/vcs.xml]: I do not think we should add the config into the repository.

Comment (KingOnTheStar) [reply .idea/vcs.xml]: Yes, I will delete it.

**Comment by gaocegege** [comment]
Still waiting for @jzp1025 's review.

**Comment by jzp1025** [comment]
LGTM , merge

**Comment by gaocegege** [comment]
@jzp1025 Please use our command /lgtm and /approve to let prow bot merge the PR automatically, thanks

**Comment by jzp1025** [comment]
@gaocegege ok , sorry for the mistake

**Comment by gaocegege** [comment]
Never mind, just a reminder :smile:

**Comment by jzp1025** [comment]
first time to review :)

**Pull request merged**